### ISSUE TITLE
Documentation does not work locally

### ISSUE THREAD

**Issue by dieselburner** [jqapi/jqapi#18]
When trying to download a [zip file](http://jqapi.com/jqapi.zip), extracting it, and opening in the Google Chrome, left menu container is empty. This is what I see in the console:

```
XMLHttpRequest cannot load file:///home/user/download/jqapi/docs/index.json. Cross origin requests are only supported for protocol schemes: http, data, chrome-extension, https, chrome-extension-resource.
```

The same is reported on [SO](http://stackoverflow.com/a/14370665/147407) in one of the comments:

```
the offline version just gives me "loading..." on the sidepanel :( –  woojoo666 Feb 11 at 0:54 
```

**Comment by wollmers** [comment]
Same here. IMHO the project is useless, if it does not deliver static HTML.

**Comment by eksperimental** [comment]
@wollmers wow.. those are harsh words.. specially when you can use Firefox, and run a simple http servers like i metioned above

**Comment by mustardamus** [comment]
Thanks @eksperimental for jumping in :).

Yep, root of the problem are browser security restrictions. @wollmers, I personally use https://www.npmjs.com/package/http-server whenever I need to quickly serve ajax powered websites locally.

P.S. I'm leaving this open, since I still have the re-worked jQAPI in the pipeline (hopefully not forever).

**tl;dr** Due to browser security restrictions ajax requests for the file:// protocol are not allowed. Use a static http server such as `python -m SimpleHTTPServer` or `npm install -g http-server && http-server .`.

**Comment by prahladyeri** [comment]
Its been almost two years. Is this bug resolved now?

**Comment by mustardamus** [comment]
jQAPI V3 is now released. Closing this issue in favor of it. Thanks.

**Comment by wollmers** [comment]
@mustardamus 
> Yep, root of the problem are browser security restrictions. @wollmers, I personally use https://www.npmjs.com/package/http-server whenever I need to quickly serve ajax powered websites locally.

I also use local webservers. But not for documentation. You either can deliver a PDF, plain text, or static HTML in the zip file, or have the docs on a public webserver. Keep it simple.

**Comment by mustardamus** [comment]
Hi @wollmers , thanks for your suggestions. I plan to use service workers to cache the pages (either all, or the ones visited).
